vars/pythonSetupPyPipeline.groovy

```diff
@@ -24,23 +24,35 @@ def call(Map pipelineParams) {
     pipelineParams["dockerFilename"] = "Dockerfile"
```

Comment:
This should probably become dockerBuildFile (to match dockerDeployFile).
Alternatively, we could have a convention and take *just* this parameter but require a suffix of ".deploy" and ".build".

Reply:
Aligned it and marked dockerFilename as deprecated